internal/config/config.go

```diff
@@ -175,10 +178,11 @@ func loadConfigFromFile(fileName string) (Config, error) {
 
 func loadConfigFromEnv(getEnv func(string) string) (Config, error) {
 	result := Config{
-		ListenAddr: getEnv(envListenAddress),
-		ServerURL:  getEnv(envServerURL),
-		Username:   getEnv(envUsername),
-		Password:   getEnv(envPassword),
+		ListenAddr:  getEnv(envListenAddress),
+		ServerURL:   getEnv(envServerURL),
+		Username:    getEnv(envUsername),
+		Password:    getEnv(envPassword),
+		TLSNoVerify: strings.ToLower(getEnv(envTLSNoVerify)) == "true",
```

Comment:
This seems to be overly strict. I'd either go with `strconv.ParseBool()` or just "all non-empty values are true".

Reply:
The intention here was to be very strict. After all, it's security option and there should be no way to "accidentally" disable checking the validity of TLS certificates.
So if someone wants to disable certificate checks, they will have to set `TLS_NO_VERIFY=true` or `TLS_NO_VERIFY=TRUE`.

Reply:
I don't think forcing the use of `true` will make it more safe, it's just more inconvenient to use. Should we use `ParseBool()` as a slightly more convenient compromise?

Reply:
I tried with strconv.ParseBool(), but now the code looks like a mess:
```go
	tlsSkipVerify := false
	if getEnv(envTLSSkipVerify) != "" {
		var err error
		tlsSkipVerify, err = strconv.ParseBool(getEnv(envTLSSkipVerify))
		if err != nil {
			return Config{}, err
		}
	}
	result := Config{
		ListenAddr:    getEnv(envListenAddress),
		ServerURL:     getEnv(envServerURL),
		Username:      getEnv(envUsername),
		Password:      getEnv(envPassword),
		TLSSkipVerify: tlsSkipVerify,
	}
```
Not sure if this is actually worth it.

Reply:
I'd say it is. For two reasons:
- Normal usage is easier because you can use more different variations of "true"
- If you use a value that is not recognizable as boolean you get a real error instead of defaulting to "false"

I have a slight variation. Mostly it produces a bit clearer error message and only calls `getEnv` once:
```go
	tlsSkipVerify := false
	if rawValue := getEnv(envTLSSkipVerify); rawValue != "" {
		value, err := strconv.ParseBool(rawValue)
		if err != nil {
			return Config{}, fmt.Errorf("can not parse value for %q: %s", envTLSSkipVerify, rawValue)
		}
		tlsSkipVerify = value
	}

	result := Config{
		ListenAddr:    getEnv(envListenAddress),
		ServerURL:     getEnv(envServerURL),
		Username:      getEnv(envUsername),
		Password:      getEnv(envPassword),
		TLSSkipVerify: tlsSkipVerify,
	}
```

Reply:
Okay, sure. In the end it's your codebase. :-)
I have updated the code according to your suggestions.

BTW, getEnv will never get called twice, the compiler will automatically figure that out by itself.